web/src/main/webapp/resources/css/atsy.css

```diff
@@ -177,10 +177,6 @@ textarea {
     margin-left: 20px;
 }
 
-#drop{
```

Comment:
Why are you delete this CSS?

Without this CSS the new application popup's position input lose its top margin.